### ISSUE TITLE
Weather effects for levels

### ISSUE THREAD

**Issue by tomangelo2** [colobot/colobot#682]
Maybe we could add some effects like sandstorm, snow, rain and similar ones? Each could have additional effects, like footprints on snow, or water dripping from objects when raining.

**Comment by krzys-h** [comment]
I remember this being suggested a loooong time ago, before we even got the source code probably. It's unlikely to happen any time soon, though.

**Comment by tomangelo2** [comment]
Yeah, I know that it's more about Beta feature, there are still many important things to do, I've just wanted to publish that idea, so it won't be forgotten.

**Comment by RaptorParkowsky** [comment]
I think this ain't going to be "normal" and "hardtextured" weather effects like other games have. I think it require some kind of low-level configurable factors in scene file. Just like particle effects in Blender where you can config gravity, velocity, randomness, etc, but here it should optimized to render real-time of course. The reason why to make it this way is possibility to create original weather scheme for every planet.

Funny trivia: Crystalium are not a snowy planet as some people on YouTube's let's play series think ;P .
